### ISSUE TITLE
Add lerna changelog

### ISSUE THREAD

**Issue by ardakkk** [mediamonks/eslint-config#35]
Currently, all packages are bumping versions manually. Use [Lerna](https://lerna.js.org/) tool to manage and publish it

**Comment by ThaNarie** [comment]
> Currently, all packages are bumping versions manually
Not really manually, it's done automatically in Github Actions. You only define if it's a major/minor/patch update, and the action does the rest.

It does however require us to publish them individually.

> Use [Lerna](https://lerna.js.org/) tool to manage and publish it
While Lerna mostly does the same, i'm not sure if it will work correctly in Github Actions.

The only benefit I see would be to run a single `lerna publish` command, and then it will bump all versions, and publish them all. But this will only work if all package versions are in sync.

Once you run in `--independent` mode, it will ask how to bump each version, which is an interactive prompt, and might not work in GH actions. Or otherwise, it's almost the same as we currently do.

Personally I don't really see the benefit of running in "sync" mode, as it links all semvers to each other (breaking change of 1 package is also a breaking change of another.

Although, a linting package doesn't really have a "public api", it mostly relies on the (peer)dependencies (e.g. supporting new parser updates, or the new eslint version), which would most likely apply to all versions at the same time.

**TLDR;**
I don't really see the need, but it also shouldn't hurt for this repo.

**Comment by ardakkk** [comment]
Regarding this comment

> Once you run in --independent mode, it will ask how to bump each version, which is an interactive prompt, and might not work in GH actions. Or otherwise, it's almost the same as we currently do.

We can set up `conventional commits` that allow bumping versions depending on commits instead of interactive prompt

**Comment by leroykorterink** [comment]
I've created a release on the releases page. I think the generated release notes are a perfectly fine, It's not necessary to change the project setup/commit conventions to generate release notes.

https://github.com/mediamonks/eslint-config/releases
https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes
